### DISCUSSION TITLE
Push Service

### DISCUSSION THREAD

**Discussion by ThorstenUm** [blakeblackshear/frigate#14983] [Ask A Question]
Hello dear specialists,

I'm looking for a way to send alarms as emails within a defined period of time. Is there a way to easily achieve this without having to use MQTT or Home Assistant? Is there a push service or similar in Frigate that calls a Phyton script when there is an alarm?

Greetings

**Comment by NickM-27** [comment]
MQTT would be the recommended way to do this, MQTT is lightweight and relatively simple to setup

**Comment by ThorstenUm** [comment]
Thanks for the info.
I tried it once.
I get the entire data list transferred via client.subscribe("frigate/events").
How can I get the "type" transferred using only client.publish("frigate/events", "type")?

Best regards and thanks

```
#!/usr/bin/env python3
import time

import paho.mqtt.client as mqtt

def on_connect(client, userdata, flags, rc):
    print("Connected with result code " + str(rc))

client = mqtt.Client()
client.on_connect = on_connect

client.username_pw_set("frigate", "frigate")
client.connect("172.28.0.2", 1883, 60)

client.loop_start()

while True:
    time.sleep(2)
    client.publish("frigate/events", "type")
```

**Reply by hawkeye217** [reply]
You need to listen on a topic, not publish to it.

**Comment by ThorstenUm** [comment]
Ah yes.
A huge data set is delivered.
Does it need to be serialized somehow?
Is there a guy here to tell me how I can access the content?

**Reply by hawkeye217** [reply]
Be sure to look at Frigate's MQTT documentation: https://docs.frigate.video/integrations/mqtt

If there are particular aspects of the object being returned that you want to look at, you'll need to do that with Python.
